Liquid/ShortDateFilter.cs

```diff
@@ -0,0 +1,20 @@
+using Fluid;
+using Fluid.Values;
+using OrchardCore.Liquid;
+using System;
+using System.Threading.Tasks;
+
+namespace Lombiq.TrainingDemo.Liquid;
+
+// This liquid filter can be used like {{ Model.ContentItem.CreatedUtc | short_date }}
+// this will render the CreatedUtc in short date format.
```

Comment:
```suggestion
// This liquid filter can be used like {{ Model.ContentItem.CreatedUtc | short_date }} this will render the CreatedUtc
// in short date format.
```